scripts/sync.py

```diff
@@ -114,6 +118,9 @@ def main():
     # load the lochness configuration file and keyring
     Lochness = config.load(args.config, args.archive_base)
 
+    # register log-file path
+    Lochness['log_file'] = str(args.log_file)
```

Comment:
Why do we need to save `args.log_file` to `Lochness` dictionary? Is the former used elsewhere to log?

Reply:
Yes, the location of log output file is used to create `s3_log.csv` in `lochness.transfer.create_s3_transfer_table`